single-node/Vagrantfile

```diff
@@ -12,6 +12,7 @@ $update_channel = "alpha"
 
 CLUSTER_IP="10.3.0.1"
 NODE_IP = "172.17.4.99"
+NODE_VCPUS = 2
```

Comment:
Did this turn out to be a requirement? It's probably generally fine to do (wouldn't work on single core machine -- not sure how common that would be). If we are keeping this, can we make this configurable in the multi-node installation too.